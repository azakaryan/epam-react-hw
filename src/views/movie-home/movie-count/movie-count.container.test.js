import React from 'react';
import { shallow } from 'enzyme';
import expectEnzymify from 'expect';
import enzymify from 'expect-enzyme';
import MovieCount, { mapStateToProps } from './movie-count.container';

expectEnzymify.extend(enzymify());


describe('Search Filters Container Component', () => {
  it('should render correctly', () => {
    const component = shallow(<MovieCount />);
    expect(component).toMatchSnapshot();
  });

  it('test mapStateToProps', () => {
    const count = mapStateToProps(new Map([['movies', []]])).count;
    expect(count).toEqual(0);
  });

  it('test mapStateToProps', () => {
    const count = mapStateToProps(new Map([['movies', [{}, {}]]])).count;
    expect(count).toEqual(2);
  });
});
