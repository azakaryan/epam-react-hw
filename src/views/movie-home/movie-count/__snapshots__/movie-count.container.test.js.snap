// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Filters Container Component should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
